product_docs/docs/transporter/ba_preview/config_reader.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: "Configuring"
+---
+
+## Config a source reader
+
+### Get credentials
+
+- Create and download a migration credentials from the postgres AI platform Migrate -> Credentials section
+- un-zip the credential and copy the whole folder to the host where the reader is installed
+```shell
+```
+
+### Configure the reader
+
+Set the following environment variables in /opt/cdcreader/run-cdcreader.sh with the right values:
+
+```shell
+### set the following environment variables:
+
+##########################################
+# Transporter Cloud Configuration        #
+##########################################
+
```

Comment:
I think this example snippet might be outdated. In my test, it includes other two lines: 
```
# Now we only support aws
#export CLOUD_PROVIDER=
```

Reply:
updated